src/libs/actions/Policy.ts

```diff
@@ -560,6 +562,114 @@ function setWorkspaceApprovalMode(policyID: string, approver: string, approvalMo
     API.write(WRITE_COMMANDS.SET_WORKSPACE_APPROVAL_MODE, params, {optimisticData, failureData, successData});
 }
 
+/**
+ *
+ * @param policyID
+ * @param reimburserEmail
+ */
+function setWorkspaceReimbursement(policyID: string, reimburserEmail: string) {
+    const policy = ReportUtils.getPolicy(policyID);
+
+    const currentPolicyReimburserEmail = policy.reimburserEmail;
+
+    const optimisticData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+            value: {
+                reimburserEmail,
+                errorFields: {reimburserEmail: null},
+                pendingFields: {reimburserEmail: CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE},
+            },
+        },
+    ];
+
+    const failureData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+            value: {
+                reimburserEmail: currentPolicyReimburserEmail,
+                errorFields: {reimburserEmail: ErrorUtils.getMicroSecondOnyxError('workflowsPage.genericErrorMessage.authorizedPayer')},
+                pendingFields: {reimburserEmail: null},
+            },
+        },
+    ];
+
+    const successData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+            value: {
+                errorFields: {reimburserEmail: null},
+                pendingFields: {reimburserEmail: null},
+            },
+        },
+    ];
+
+    const params: SetWorkspaceReimbursementParams = {policyID, reimburserEmail};
+
+    API.write(WRITE_COMMANDS.SET_WORKSPACE_REIMBURSEMENT, params, {optimisticData, failureData, successData});
+}
+
+/**
+ * clear the workspace reimbursement error while setting the authorized payer
+ * @param policyID
+ */
+function clearWorkspaceReimbursementError(policyID: string) {
+    Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, {errorFields: {reimburserEmail: null}});
+}
+
+/**
+ * set the workspace reimbursement choice
+ * @param policyID
+ * @param reimbursementChoice
+ */
+function setReimbursementFromChoice(policyID: string, reimbursementChoice: ValueOf<typeof CONST.POLICY.REIMBURSEMENT_CHOICES>) {
```

Comment:
I think the command name specified in the doc is `SetWorkspaceReimbursement`, am I missing something?

Reply:
@lakchote can you take a look at these command names? The doc specifies `SetWorkspaceReimbursement` and `WorkspaceWorkflowsPayer`, but I see that the API implemented `SetReimbursementFromChoice` and nothing for the payer (that I could find)